### ISSUE TITLE
Include DeepL Translator

### ISSUE THREAD

**Issue by ohager** [gilmarsquinelato/i18n-manager#322]
Would be nice, if the DeepL Translator API could be integrated. The translation quality of DeepL outperforms Google Translate.

https://www.deepl.com/docs-api.html

**Comment by gilmarsquinelato** [comment]
Hey @ohager,

Indeed DeepL sounds to be way better,
Right now I'm developing the v3 of i18n Manager, and this release have some big implementations, maybe this feature can be put in the v3.1 :)

**Comment by Androz2091** [comment]
It would be very very nice!

**Comment by daniol** [comment]
The Translate API of Yandex produces very good results for most language sets, it is also free up to 1.000.000 chars per day and uses a very easy syntax.

Info: https://tech.yandex.com/translate/
API reference: https://tech.yandex.com/translate/doc/dg/reference/translate-docpage/

**Comment by Androz2091** [comment]
I'm not sure the quality of Yandex Translate API is the same as the quality of DeepL...

**Comment by gilmarsquinelato** [comment]
Indeed, I don't know it Yandex one is good enough to be used now, it doesn't even put the customers using it.

**Comment by daniol** [comment]
I just used Yandex last week to translate about 400 strings of a website and it worked well for spanish, catalan and german.

**Comment by gilmarsquinelato** [comment]
Cool, but we don't have more use cases/companies using it.
This is because we don't know how long it is at the market, breaking changes, etc.

**Comment by gilmarsquinelato** [comment]
At the other hand, we have DeepL, that seems to be longer in the market.
